controllers/watermarkpodautoscaler_controller.go

```diff
@@ -217,70 +225,70 @@ func (r *WatermarkPodAutoscalerReconciler) reconcileWPA(ctx context.Context, log
 	reference := fmt.Sprintf("%s/%s/%s", wpa.Spec.ScaleTargetRef.Kind, wpa.Namespace, wpa.Spec.ScaleTargetRef.Name)
 	setCondition(wpa, autoscalingv2.AbleToScale, corev1.ConditionTrue, datadoghqv1alpha1.ConditionReasonSuccessfulGetScale, "the WPA controller was able to get the target's current scale")
 
-	metricStatuses := wpaStatusOriginal.CurrentMetrics
-	proposedReplicas := int32(0)
-	metricName := ""
-
 	desiredReplicas := int32(0)
 	rescaleReason := ""
 	now := time.Now()
-	rescale := true
+	var rescale bool
```

Comment:
the `rescale` instantiation can be done line 262 https://github.com/DataDog/watermarkpodautoscaler/pull/187/files#diff-70d5250d0b22ca57247df02274172975492cd9862cc800a40ec91644a462d9ddR262